tests/nodeos_trust_evm_test.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+import random
+import os
+import json
+
+import rlp
+from ethereum import transactions
+from binascii import unhexlify
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.TestHelper import AppArgs
+from core_symbol import CORE_SYMBOL
+
+
+###############################################################
+# nodeos_trust_evm_test
+#
+# Set up a TrustEVM env and run simple tests.
+#
+# Need to install:
+#   rlp       - pip install rlp
+#   ethereum  - pip install ethereum
+#          venv/lib/python3.6/site-packages/ethereum/transactions.py
+#          /home/account/.local/lib/python3.8/site-packages/ethereum/transactions.py
+#          Line 135
+#
+#          135c135
+#          <             self.v += 8 + network_id * 2
+#          ---
+#          >             v += 8 + network_id * 2
+#
+# --turst-evm-root should point to the root of TrustEVM build dir
+# --trust-evm-contract-root should point to root of TrustEVM contract build dir
+#
+# Example:
+#  ./tests/nodeos_trust_evm_test.py --trust-evm-contract-root ~/ext/TrustEVM/contract/build --leave-running
+#
+#  Launches wallet at port: 9899
+#    Example: bin/cleos --wallet-url http://127.0.0.1:9899 ...
+#
+###############################################################
+
+Print=Utils.Print
+errorExit=Utils.errorExit
+
+appArgs=AppArgs()
+extraArgs = appArgs.add(flag="--trust-evm-root", type=str, help="TrustEVM build dir", default="")
+extraArgs = appArgs.add(flag="--trust-evm-contract-root", type=str, help="TrustEVM contract build dir", default="")
+
+args=TestHelper.parse_args({"--keep-logs","--dump-error-details","-v","--leave-running","--clean-run" }, applicationSpecificArgs=appArgs)
+debug=args.v
+killEosInstances= not args.leave_running
+dumpErrorDetails=args.dump_error_details
+keepLogs=args.keep_logs
+killAll=args.clean_run
+trustEvmRoot=args.trust_evm_root
+trustEvmContractRoot=args.trust_evm_contract_root
```

Comment:
Looks like this is required below. Should a good default be provided or else should you remove the default in the args and make sure the caller provides a good path?